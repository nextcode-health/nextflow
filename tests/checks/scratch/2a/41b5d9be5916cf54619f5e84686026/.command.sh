#!/bin/bash -ue
echo 'x: 2; y: Ciao; z: beta'
