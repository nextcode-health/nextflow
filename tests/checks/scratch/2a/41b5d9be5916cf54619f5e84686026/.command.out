x: 2; y: Ciao; z: beta
